src/collections/_documentation/platforms/php/index.md

```diff
@@ -0,0 +1,159 @@
+---
+title: PHP
+sidebar_order: 3
+---
+
+{% include learn-sdk.md platform="php" %}
+
+The [Sentry PHP SDK](https://packagist.org/packages/sentry/sentry) provides
+support for PHP 7.1 or later.
+
+This documentation goes over some PHP specific things such as integrations to
+frameworks.
+
+## Integrations
+
+*Integrations* extend the functionality of the SDK for some common frameworks and
+libraries.  They can be seen as plugins that extend the functionality of the Sentry SDK.  
+Integrations are configured by a call to `init()`.
+
+## PHP Specific Options
+
+PHP provides some additional options, all options can be passed into `init()`.
+
+#### send_attempts
+
+The number of attempts that should be made to send an event before erroring
+and dropping it from the queue.
+By default this option is set to `6`.
+
+#### project_root
+
+The root of the project source code. As Sentry is able to distinguish project
+files from third-parties ones (e.g. vendors), this option can be configured to
+mark the directory containing all the source code of the application.
+
+For example, assuming that the directory structure shown below exists, marking
+the project root as ``project-folder/src/`` means that every file inside that
```

Comment:
Use a single backtick instead of two (I believe this is a misprint from the reStructuredText documentation)